src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -178,7 +182,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             DockerAccess access = null;
             try {
                 access = createDockerAccess(minimalApiVersion);
-                ServiceHub serviceHub = serviceHubFactory.createServiceHub(project, session, access, log, logSpecFactory);
+                fetchLimit = resolvedImages.get(0).getRunConfiguration().getFetchLimit() != null
```

Comment:
Cool, filtering with 'ancestor=<image name>' works. Unfortunately only available since 1.23

But with this two features we should get rid of the local iteration of image names (we leave it in nevertheless so that the plugin will work with older clients, too).